drivers/gpio/Kconfig.xmc4xxx

```diff
@@ -0,0 +1,13 @@
+# XMC4XXX GPIO configuration
+#
+# Copyright (c) 2022 Schlumberger
+# SPDX-License-Identifier: Apache-2.0
+
+DT_COMPAT_INFINEON_XMC4XXX_GPIO := infineon,xmc4xxx-gpio
```

Comment:
Look at new Kconfig style in https://github.com/zephyrproject-rtos/zephyr/pull/48085